src/Styleguide/Components/Lightbox.tsx

```diff
@@ -0,0 +1,213 @@
+import { themeProps } from "@artsy/palette"
+import { bind, once, throttle } from "lodash"
+import React from "react"
+import ReactDOM from "react-dom"
+import styled from "styled-components"
+import { themeGet } from "styled-system"
+import { Flex } from "Styleguide/Elements/Flex"
+import { Slider, SliderProps } from "./LightboxSlider"
+
+const KEYBOARD_EVENT = "keyup"
+
+const DeepZoomContainer = styled.div`
+  position: fixed !important;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  z-index: 1000;
+  transition: opacity 0.5s;
+  background-color: ${themeGet("colors.black100")};
+`
+
+export interface LightboxProps {
+  deepZoom: {
+    Image: {
+      xmlns: string
+      Url: string
+      Format: string
+      Overlap: number
+      TileSize: number
+      Size: {
+        Width: number
+        Height: number
+      }
+    }
+  }
+}
+
+export interface LightboxState {
+  shown: boolean
+  element: Element
+  viewer: any
+  deepZoomRef: any
+  slider: SliderProps
+  promisedDragon: Promise<any>
+}
+
+export class Lightbox extends React.Component<LightboxProps, LightboxState> {
+  state = {
+    element: null,
+    viewer: null,
+    shown: false,
+    deepZoomRef: React.createRef(),
+    slider: {
+      min: 0,
+      max: 1,
+      step: 0.01,
+      value: 0,
+    },
+    promisedDragon: import("openseadragon"),
```

Comment:
Rather than use new `import` syntax which we still need to verify across the force / reaction boundary can this be revised to just `require('openseadragon')` with a TODO?

Reply:
I can do that. It'll change the load time performance of the artwork entry chunk though.

Reply:
We should probably worry about performance impacts once some open questions around force / reaction interop have been answered and more importanly until we measure.